docs/events/jarsigning.md

```diff
@@ -0,0 +1,83 @@
+Signing a JAR
+=============
+
+Forge allows Mods to sign their jar files with a signature. However, these signatures are not used
+as security and shouldn't be used this way. Signatures are used as sanity checks, so that Modders
+are able to check if he is running his own un-edited code.
+
+!!! note
+
+	If someone really wants to get around this system, they can. This system is not one hundred percent effective. This system is event based, meaning that an invalid key will cause an event to be executed.
```

Comment:
I'd reword this paragraph (and the sentence about events does not belong here at all):
Once again keep in mind that this system is not a security measure. With enough malicious intend it can be circumvented.